### PR TITLE
remove dead code

### PR DESCRIPTION
**Pull request by komuw**
# what have you changed?
- removed dead code

# why
- fixes https://github.com/gomods/athens/issues/232

notes:
- i mostly used `https://github.com/dominikh/go-tools/tree/master/cmd/unused` to find dead code

### CODE REVIEW

File: cmd/proxy/actions/cdn.go [hunk 1]
```diff
@@ -1,10 +1 @@
 package actions
```
Comment (robjloranger) [inline cmd/proxy/actions/cdn.go]: This file will be empty and can be removed completely. I forget if there was a use still for getting fake metadata. I know this function is not used anywhere currently, just wondering if someone needed it.

**Comment by robjloranger** [comment]
also once you feel your PR is complete you can remove the `[WIP]` to signal that it is done.

**Comment by arschles** [comment]
@komuw can you resolve the conflicts in this and ping me when you're done? I'll merge right away then

**Comment by arschles** [comment]
@komuw a few test failures to fix. when you do the fixes, let me know and I'll do another quick review + merge 😄

**Comment by robjloranger** [comment]
@arschles, there were some tests being used while @tbpg was fixing up the (./pkg/storage).Deleter error returns which will be moved into the common suite after which some of the issues should clear up.

**Comment by robjloranger** [comment]
Great work @komuw

**Pull request merged**